### ISSUE TITLE
Test server tether-io.dev should run Eclipse MQTT

### ISSUE THREAD

**Issue by anselanza** [RandomStudio/tether#23]
RabbitMQ is not compliant with a lot of MQTT features, including:
- retained messages
- invalid wildcard subscriptions

**Comment by anselanza** [comment]
Since it's now so easy to get MQTT up and running using a single Docker command.... do we even need this server?

**Comment by ralphkok** [comment]
I don't think we do. I've not used it for ages anyway. I'm fine with deprecating it, including everything we have on there (i.e. the older tether packages)

**Comment by anselanza** [comment]
Let's delete!
